build.py

```diff
@@ -6,6 +6,6 @@
 
 if __name__ == "__main__":
 
-    builder = build_template_default.get_builder()
+    builder = build_template_default.get_builder(build_policy = "missing")
```

Comment:
Why do you need it? Is there any package missing?

Reply:
Binary package of mbedtls/2.6.1@bincrafters/stable isn't available for some latest compilers (clang 9, GCC 7 and 8, VS19 and apple-clang 10).

Reply:
understood, I'll provide it